torch/__init__.py

```diff
@@ -482,7 +482,7 @@ def sym_min(a, b):
 
 for name in dir(_C):
     if name[0] != '_' and not name.endswith('Base'):
-        __all__.append(name)
+        __all__ += list(name)
```

Comment:
Wouldn't it case the str to a list and append it character by character? What you actually want is 
```suggestion
        __all__ += [name]
```
Either way, I doubt the type checker will be able to figure out the complex loop logic without actually running it.

Reply:
thanks I updated it! I originally used this(very wrong solution :sweat_smile:  ) in case of the PLE0605 error the change introduced but I should've just used `#noqa: PLE0605` like this:

https://github.com/pytorch/pytorch/blob/2b2b6caf8f4da5599841678967202fb9443c56e5/torch/_numpy/_dtypes.py#L409-L422